x-pack/plugins/cases/public/components/create/flyout/create_case_flyout.tsx

```diff
@@ -26,6 +26,7 @@ export interface CreateCaseFlyoutProps {
   onSuccess?: (theCase: Case) => Promise<void>;
   attachments?: CaseAttachmentsWithoutOwner;
   headerContent?: React.ReactNode;
+  initialValue?: Partial<Case>;
```

Comment:
The `Case` contains attributes that are not part of the allowed properties of the post request. We should make the type as strict as possible. What about `initialValue?: Pick<CasePostRequest, 'title' | 'description'>;`? This way we can be sure that when we spread in the form `defaultValue` only allowed values will be added.